MetaMask/metamask-extension
# UX: Add event tracking for empty banner displays

## **Description**

Adds event tracking for the new "Buy", "Receive", and "NFT" banners.

## **Related issues**

Fixes: https://github.com/MetaMask/MetaMask-planning/issues/1949

## **Manual testing steps**

1. Create a new wallet (will have 0 balance, 0 NFTs)
2. Open `ui/contexts/metametrics.js` and in the `trackEvent` function, add a `console.log` to see the events get recorded (or use Logpoints in the debugger)

## **Screenshots/Recordings**

### **Before**

N/A

### **After**

N/A

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
